index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>
+        Hello Mucroverse
+    </h1>
```

Comment:
Great job @Hassaanjbaig-code 🤝

- [ ] As stated in the project requirements, kindly add a class to your HTML element and apply styles to this element through this class in your stylesheet.